kratos/utilities/variable_redistribution_utility.cpp

```diff
@@ -12,115 +12,316 @@
 //
 
 #include "includes/fsi_variables.h"
+#include "utilities/atomic_utilities.h"
+#include "utilities/parallel_utilities.h"
 #include "variable_redistribution_utility.h"
 
 namespace Kratos
 {
 
+// Historical API
+// ConvertDistributedValuesToPoint for double variables
+KRATOS_DEPRECATED_MESSAGE("This ConvertDistributedValuesToPoint() signature is deprecated. Use the one with Elements/Conditions container.")
```

Comment:
```suggestion
```

not even sure this works in the cpp

In any case I would remove it in this case (same for the other methods)

Reply:
OK. Removing then.